cypress/support/commands.js

```diff
@@ -11,12 +11,14 @@ Cypress.Commands.add('autologin', () => {
     password = 'secret';
   }
 
-  cy.request({
-    method: 'POST',
-    url: `${api_url}/@login`,
-    headers: { Accept: 'application/json' },
-    body: { login: user, password: password },
-  }).then(response => cy.setCookie('auth_token', response.body.token));
+  return cy
```

Comment:
@sneridagh just out of curiosity. Could you elaborate why returning cy.request here makes a difference?

Reply:
It seems that Cypress expects you to return (the promise) so he can know when it’s resolved better.